<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KeyboardShortcuts_Dialog</class>
 <widget class="QDialog" name="KeyboardShortcuts_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>292</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>449</width>
    <height>292</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>449</width>
    <height>292</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Keyboard shortcuts</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Existing shortcuts:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="_100">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>411</width>
     <height>181</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="_101">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>230</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Key</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="_102">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Ctrl</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="_103">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>230</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Alt</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_1">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>230</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="_104">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
